meshroom/ui/qml/GraphEditor/AttributeItemDelegate.qml

```diff
@@ -118,32 +120,40 @@ RowLayout {
         }
     }
 
+    function setAttribute(value) {
```

Comment:
Would be better to emit a signal and react to the event outside depending on the context.